manifests/03-rbac-role-cluster.yaml

```
@@ -67,3 +67,16 @@ rules:
   - create
   - update
   - delete
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-crd-reader
```

I'd just call this `console` so that we could add additional permissions later.

```suggestion
  name: console
```